source/includes/fact-wiredtiger-cachesizegb.rst

```diff
@@ -0,0 +1,12 @@
+The default value for this setting is optimized for a single :program:`mongod`
+instance per node. If a single node contains multiple instances, then you should
+change :setting:`storage.wiredTiger.engineConfig.cacheSizeGB` for each instance
+to a value that is large enough to encompass the entire working dataset,
+but that also leaves room for the other :program:`mongod` instances.
```

Comment:
little wordy.

you can play with something like:
 ... for each instance, adjust the :setting:`~storage.wiredTiger.engineConfig.cacheSizeGB` setting to accommodate the working set as well as the other :program:`mongod` instances.